tchannel/testing/vcr/patch.py

```diff
@@ -103,6 +103,7 @@ def send(self, arg1, arg2, arg3,
             vcr_response_future = tchannel.thrift(
                 proxy.VCRProxy.send(vcr_request),
                 hostport=self.vcr_hostport,
+                timeout=1.0,
```

Comment:
- you'll want this to be configurable by those using vcr
- seems like a low default